IO.go

```diff
@@ -295,6 +295,15 @@ func (io IO) SetBackendFlags(flags BackendFlags) {
 	C.iggIoSetBackendFlags(io.handle, C.int(flags))
 }
 
+// SetMouseDrawCursor request ImGui to draw a mouse cursor for you (if you are on a platform without a mouse cursor).
+func (io IO) SetMouseDrawCursor(show bool) {
+	var showArg C.IggBool
```

Comment:
For future reference: There's a `castBool()` helper that does this block.

Reply:
Nice! I just looked around for a random other function that used bool and must have stumbled upon one of the few that isn't using the helper: https://github.com/inkyblackness/imgui-go/blob/main/IO.go#L129